.github/workflows/pkgci_regression_test_amdgpu_vulkan.yml

```diff
@@ -69,4 +69,4 @@ jobs:
           IREE_TEST_CONFIG_FILES: experimental/regression_suite/external_test_suite/config_gpu_vulkan.json
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest SHARK-TestSuite/iree_tests -n 4 -rpfE --timeout=60
+          pytest SHARK-TestSuite/iree_tests -n 4 -rpfE --timeout=30 --retries=2
```

Comment:
https://github.com/openxla/iree/actions/runs/8209267488/job/22454665007?pr=16718#step:9:48

argh... so close - missing the actual name of the test case in this message
```
======================= the following tests were retried =======================
	gpu_vulkan failed on attempt 1! Retrying!
	Traceback (most recent call last):
	  File "/_work/iree/iree/venv/lib/python3.11/site-packages/_pytest/outcomes.py", line 188, in fail
	    raise Failed(msg=reason, pytrace=pytrace)
	Failed: Timeout >30.0s

	gpu_vulkan passed on attempt 2!

...

============ 313 passed, 918 xfailed, 1 retried in 61.84s (0:01:01) ============
```

Reply:
(Test names and source paths are configured in a way I'm not totally happy with yet - e.g. `onnx/node/generated/test_clip_splitbounds/model.mlir::gpu_vulkan` and `onnx/node/generated/test_clip_splitbounds/model.mlir::cpu_llvm_sync`, but just the trailer got used in that message)